tests/windows_test.go

```diff
@@ -69,9 +69,10 @@ var _ = Describe("[Serial][sig-compute]Windows VirtualMachineInstance", func() {
 		util.PanicOnError(err)
 		checks.SkipIfMissingRequiredImage(virtClient, tests.DiskWindows)
 		libstorage.CreatePVC(OSWindows, "30Gi", libstorage.Config.StorageClassWindows, true)
-		windowsVMI = tests.NewRandomVMI()
+		windowsVMI = libvmi.New()
 		windowsVMI.Spec = getWindowsVMISpec()
 		tests.AddExplicitPodNetworkInterface(windowsVMI)
+		windowsVMI.Namespace = util.NamespaceTestDefault
```

Comment:
Please remove this.

Reply:
The reason we need this is because `tests.NewRandomVMI()` set the namespace of the vmi to be `util.NamespaceTestDefault`, the tests rely on the fact that the vmi's namespace is set.

For instance:
We have functions that uses vmi.namespace such as `tests.WaitForSuccessfulVMIStartWithTimeout` 
in internal call to  `waitForVMIPhase`
https://github.com/kubevirt/kubevirt/blob/52a3f52ee14c14a6cc58f5bb16ebb8784d686c9a/tests/utils.go#L1062

and we call `tests.WaitForSuccessfulVMIStartWithTimeout` tests.WaitForSuccessfulVMIStartWithTimeout in:
https://github.com/kubevirt/kubevirt/blob/52a3f52ee14c14a6cc58f5bb16ebb8784d686c9a/tests/windows_test.go#L100